test/performance/logs/agent_config.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Adding MOM metrics for the agent to self-monitoring and having the same configuration with the [current logs performance testing](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/test/performancetest/resources/config.json)